IcedTea OpenJDK 3.11.0 stops static linking with NSS, breaking our builds

See: https://bugs.alpinelinux.org/issues/10126
Also see: https://github.com/docker-library/openjdk/issues/289

You will get errors like the following near the end of your error log:

```
Caused by: java.io.FileNotFoundException: /usr/lib/libnss3.so
	at sun.security.pkcs11.Secmod.initialize(Secmod.java:193)
	at sun.security.pkcs11.SunPKCS11.<init>(SunPKCS11.java:218)
```

--- a/dind/Dockerfile
+++ b/dind/Dockerfile
@@ -1,7 +1,7 @@
 FROM docker:latest
 LABEL maintainer="zephinzer <dev-at-joeir-dot-net>" \
       description="Docker-in-Docker container for use in CI pipelines"
-ENV APK_DEPENDENCIES="bash curl git jq vim openssh build-base python-dev python py-pip openjdk8 mysql-client" \
+ENV APK_DEPENDENCIES="bash curl git jq vim openssh build-base python-dev python py-pip openjdk8 mysql-client nss" \
     PIP_DEPENDENCIES="docker-compose" \
     PATHS_TO_REMOVE="\
       /usr/include/* \

